pyximport/pyximport.py

```diff
@@ -300,19 +301,31 @@ def find_module(self, fullname, package_path=None):
             paths = package_path
         else:
             paths = sys.path
-        join_path = os.path.join
-        is_file = os.path.isfile
-        is_abs = os.path.isabs
-        abspath = os.path.abspath
```

Comment:
The boundary between "too verbose" and "not too verbose" is often subtle. To me the extra characters `os.`  or `os.path` add are acceptable. Moreover, the rest of the file uses `os.path` directly without any aliases, so I think it is best to make this bit consistent.